### ISSUE TITLE
Spinner is sometimes not removed from console output

### ISSUE THREAD

**Issue by klaussner** [meteor/meteor#10571]
Since Meteor 1.8.1, the last spinner character is sometimes not removed when the output moves to the next line. To reproduce the bug, run any Meteor project:

```
$ meteor
[[[[[ ~/spinner-test ]]]]]                 

=> Started proxy.                             
=> Started MongoDB.                           
=> Started your app.                         \

=> App running at: http://localhost:3000/
```

The `\` next to `=> Started your app.` is not removed (tested on Mac and Linux). I think this is related to the commits https://github.com/meteor/meteor/commit/a985bf4b22f19d0ed1d6e2f96c6ab2331983565b and https://github.com/meteor/meteor/commit/f9d17aaf0fcfee9857ec6390a5cfc5f277b88d57 but I haven't had time to take a closer look yet.

cc @zodern 🙂

**Comment by zodern** [comment]
This will be fixed in 1.8.2 by https://github.com/meteor/meteor/pull/10523

**Comment by klaussner** [comment]
I completely overlooked that PR. 🤦‍♂ Thanks! 🙂
